crud/common/utils.lua

```diff
@@ -156,4 +156,20 @@ function utils.convert_operations(user_operations, space_format)
     return converted_operations
 end
 
+function utils.unflatten_rows(rows, metadata)
```

Comment:
Why not `utils.unflatten_rows({ rows = ..., metadata = ...})`? 

CRUD operations return exactly `{ rows = ..., metadata = ...}` table.

Reply:
I plan it will be a general purpose function.

And your suggestion is a restriction of it for specific task. If you have tuples and space:format() locally you can simply call this function without any formatting `unflatten(tuples, space:format())`.

Reply:
In this case second argument is `space_format`, not metadata?

Reply:
Well, I'll rename

Reply:
However, no... No. In future we allow to specify set of fields that should be fetched from storage.

space:format() - it's a case of metadata and currently (not in future) it's the same as metadata.

Reply:
Yes, I agree,